Bottom: 0df9bfaeac1657018b37f26031005e769e2fc343
Top:    eacdeb901ed178384e83ecc14bc1e14308823c65
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 15:57:39 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/provision.sh b/provision.sh
new file mode 100755
index 0000000..cf996dc
--- /dev/null
+++ b/provision.sh
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+#PYTHONUNBUFFERED=1 \
+     
+
+ANSIBLE_FORCE_COLOR=true \
+ANSIBLE_HOST_KEY_CHECKING=false \
+ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ControlMaster=auto -o ControlPersist=60s' \
+exec ansible-playbook \
+        --connection=ssh \
+        --timeout=30 \
+        --limit="vmdaedalus.novatech-llc.com" \
+        --inventory-file=/home/coopera/Developer/testdaedalus/vmdaedalus/vagrant_ansible_inventory \
+        -v \
+        ansible-playbook/site.yml "$@"
